restli-server/src/main/java/com/linkedin/restli/internal/server/model/RestLiAnnotationReader.java

```diff
@@ -1978,6 +1980,21 @@ private static void addResourceMethods(final Class<?> resourceClass,
     validateResourceModel(model);
   }
 
+  static int getMethodIndex(Method method)
```

Comment:
nit: Maybe your comment about the motivation for sorting batchFinders before finders should be left as a javadoc here so this method itself is clear. other than this LGTM